src/getDataset.js

```diff
@@ -54,25 +54,24 @@ const requestV1Dataset = async (metaJsonUrl, treeJsonUrl) => {
  * then the promise will reject.
```

Comment:
P.S. the comment immediately preceding the function is now out-of-date with this PR